src/utils/config_manager.ts

```diff
@@ -0,0 +1,13 @@
+// tslint:disable: prefer-function-over-method
+import { RFQM_API_KEY_WHITELIST } from '../config';
+
+/**
+ * ConfigManager is a simple wrapper around configs.
+ *
+ * It exists to provide a layer around our configs, which can then be mocked while writing tests
+ */
+export class ConfigManager {
```

Comment:
I'm curious why you ended up creating a class here? It looks like the class has no local state. Have you considered simply using the whitelist constant in the code without `ConfigManager`?

Reply:
I'm not opposed to it, I just noticed that there isn't any state on this class and therefore I am curious if we could simplify things by removing this class/file. Maybe I'm not seeing things right? Let me know your opinion.

Reply:
I added this class to mock out `RFQM_API_KEY_WHITELIST` - otherwise, I would need to gain access to the environment variables before the tests run, which is tricky - not to mention a bit hard to read for future test maintainers.

In other words, this class only exists for testability. Otherwise, I do agree, it's not particularly useful

Reply:
that's a good reason! it's fine